android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -2010,6 +2012,32 @@ public static int getOrientation(Context context) {
     return Configuration.ORIENTATION_UNDEFINED;
   }
 
+  /**
+   * Get the long-press delay (in milliseconds)
+   * @param context
+   * @return int - long-press delay in milliseconds
+   */
+  public static int getLongpressDelay(Context context) {
+    int defaultDelay = 500; // default longpress delay in KeymanWeb (ms)
```

Comment:
This should be a global constant somewhere rather than buried in this function

Reply:
Done in #12185 KMManager
```
public static final int KMDefault_LongpressDelay = 500;
```